backend/app/controllers/step_subtask.py

```diff
@@ -20,7 +20,7 @@ def analysis():
     username = session['username']
     requirementDoc = session[username]['memory']['originalPrompt']
     sourceBranch = session[username]['memory']['task_info']['source_branch']
-    wsPath = get_ws_path(session[username]['memory']['task_info']['task_id'])
+    _ = get_ws_path(session[username]['memory']['task_info']['task_id'])
```

Comment:
Thank you. Just delete the unused variables

Reply:
ok，good idea，got it

Reply:
> Thank you. Just delete the unused variables

done @booboosui